frontend/apps/reader/modules/readerscrolling.lua

```diff
@@ -339,14 +340,14 @@ function ReaderScrolling:_setupAction()
                 self._last_manual_scroll_dy = 0
                 return false
             end
-            if self._last_manual_scroll_duration:isZero() or self._last_manual_scroll_dy == 0 then
+            if self._last_manual_scroll_duration == 0 or self._last_manual_scroll_dy == 0 then
                 return false
             end
 
             -- Initial velocity is the one of the last pan scroll given to accountManualScroll()
-            local delay = self._last_manual_scroll_duration:tousecs()
-            if delay < 1 then delay = 1 end -- safety check
-            self._velocity = self._last_manual_scroll_dy * 1000000 / delay
+            local delay_us = time.to_us(self._last_manual_scroll_duration)
+            if delay_us < 1 then delay_us = 1 end -- safety check
+            self._velocity = self._last_manual_scroll_dy * time.s(1.000000 / delay_us)
```

Comment:
You could let `1.00000` be `1`, Lua (unlike Python2) won't do integer arithmetic if both values are integer).

Reply:
Yes, writine `1` or `1.0` or `1.00000` doesn't make a difference.